data/json/items/melee/spears_and_polearms.json

```diff
@@ -365,7 +365,7 @@
     "to_hit": { "grip": "solid", "length": "short", "surface": "point", "balance": "neutral" },
     "price": "2 USD",
     "price_postapoc": "10 cent",
-    "weapon_category": [ "FENCING_WEAPONRY" ],
+    "weapon_category": [ "SPEARS" ],
```

Comment:
```suggestion
```
It's too short to be a spear at 60cm or less than 2 feet.  I think it's fine for some makeshift weapons to not have a category at all.  I agree with you that it's not really an appropriate fencing weapon.